docs/sources/community/maintaining/release/_index.md

```diff
@@ -12,23 +12,32 @@ to publish a new [Grafana Loki](https://github.com/grafana/loki) release.
 
 ## Release stable version
 
-1. [Create release branch]({{< relref "./1-create-release-branch" >}})
-1. [Backport commits]({{< relref "./2-backport-commits" >}})
-1. [Prepare Release notes]({{< relref "./3-prepare-release-notes" >}})
-1. [Prepare Changelog]({{< relref "./4-prepare-changelog" >}})
-1. [Document Metrics and Configurations changes]({{< relref "./5-document-metrics-configurations-changes" >}})
-1. [Prepare Upgrade guide]({{< relref "./6-prepare-upgrade-guide" >}})
-1. [Update version numbers]({{< relref "./7-update-version-numbers" >}})
-1. [Tag Release]({{< relref "./8-tag-release" >}})
-1. [Publish Release]({{< relref "./9-publish-release" >}})
+1. [Create release branch]({{< relref "./create-release-branch" >}})
```

Comment:
When I click "View file" in the options above I expect the `relref`s to be rendered correctly as links, but they don't. Are we sure removing the numbers don't break it?

Reply:
I think `relref` is a hugo template that get rendered on grafana website. I don't think it renders correctly on Github AFAIK.

e.g: similar relref on other pages https://github.com/grafana/loki/blob/cd7c40f9eea535fdcba1428af617b8e303085c6d/docs/sources/get-started/_index.md